webapp/src/components/code_comments/code_comments.tsx

```diff
@@ -0,0 +1,121 @@
+import { Client4 } from 'mattermost-redux/client';
+import { Post } from 'mattermost-redux/types/posts';
+import React from 'react';
+
+export type Props = {
+    currentUserId: string;
+    channel: string;
+    thread: string;
+    file: string;
+    line?: number;
+    comments: Post[];
+    getLineContext: (n: number) => string
+};
+
+type State = {
+    visible: boolean;
+    draftValue: string;
+    contextLines: number;
+};
+
+const Avatar = (window as any).Components.Avatar;
+const Textbox = (window as any).Components.Textbox;
+const imageURLForUser = (window as any).Components.imageURLForUser;
+const formatText = (window as any).PostUtils.formatText;
```

Comment:
I think these types can be used here

```suggestion
const Avatar = (window as any).Components.Avatar as React.ElementType;
const Textbox = (window as any).Components.Textbox as React.ElementType;
const imageURLForUser = (window as any).Components.imageURLForUser as (userId: string) => string;
const formatText = (window as any).PostUtils.formatText as (text: string) => string;
```